phone-scripts/02-install-packages.sh

```diff
@@ -27,13 +27,13 @@ dnf remove kernel || rpm -e --noscripts kernel-core
 dnf install linux-firmware
 
 infecho "Installing kernel..."
-dnf --disablerepo="*" --enablerepo="copr:copr.fedorainfracloud.org:njha:mobile" install kernel
+dnf --disablerepo="*" --enablerepo="copr:copr.fedorainfracloud.org:njha:mobile" install megi-kernel
```

Comment:
Install the megi-kernel from here: https://gitlab.com/fedora-mobile/kernel-megi

I also uploaded the boot.scr to use in the scripts.